src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -110,4 +116,26 @@ def test_email_sent(self):
 
         email = mail.outbox[0]
         self.assertEqual(email.to, ["hello@open-forms.nl"])
-        self.assertEqual(email.subject, _("Your form submission"))
+        self.assertEqual(email.subject, _(f"Saved form {submission.form.name}"))
+
+        self.assertIn(submission.form.name, email.body)
+        self.assertIn("15-11-2021", email.body)
+
+        submission.refresh_from_db()
+        token = submission_resume_token_generator.make_token(submission)
+        resume_path = reverse(
+            "submissions:resume",
+            kwargs={
+                "token": token,
+                "submission_uuid": submission.uuid,
+            },
+        )
+        expected_resume_url = urljoin("http://testserver", resume_path)
+
+        self.assertIn(expected_resume_url, email.body)
+
+        datetime_removed = submission.created_on + timedelta(
+            days=GlobalConfiguration.get_solo().incomplete_submissions_removal_limit
+        )
+
+        self.assertIn(datetime_removed.date().strftime("%d-%m-%Y"), email.body)
```

Comment:
Please use literal values. Set up your configuration in the test, calculate the planned removal date and compare that that value is in the result. You can use `django.template.defaulttags.date` filter function here though, but define `datetime_removed = datetime(...)` in that case.